packages/nvidia_nat_app/src/nat_app/graph/llm_detection.py

```
@@ -0,0 +1,363 @@
+                    found[f"{name}.{attr_name}"] = attr_val
+
+        if callable(obj) and not detector.is_llm(obj):
+            try:
+                hints = typing.get_type_hints(obj)
+                ret = hints.get("return")
+                if ret is not None and isinstance(ret, type):
+                    sentinel = object.__new__(ret) if not inspect.isabstract(ret) else None
+                    if sentinel is not None and detector.is_llm(sentinel):
+                        found[name] = obj
+            except Exception:  # pylint: disable=broad-exception-caught
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)